unlock-js/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.3.11
 - add `getTokenSymbol` method to web3Service to identify arbitrary ERC20 tokens (#4481)
 - add `getTokenBalance` method to web3Service to get the user's balance of arbitrary ERC20 tokens (#4431)
+- fixed a difference between pending transactions (node knows about them) and submitted transactions (which may be transactions that have been cancelled and will never succeed)
```

Comment:
We should put this in a `next version` field since we published 0.3.11 this afternoon!